src/config/constants.js

```diff
@@ -0,0 +1,15 @@
+require('dotenv').config()
+const Web3 = require('web3');
+
+const slackUrl = process.env.SLACK_WEBHOOK_URL;
+const networkName = process.env.NETWORK_NAME;
+const urlExplorer = networkName == "rinkeby" ? 'rinkeby.etherscan.io' : 'etherscan.io';
```

Comment:
clarify in the Readme that this bot is aimed for Mainnet or rinkeby, not other networks